.github/dependabot.yml

```diff
@@ -1,6 +1,7 @@
 version: 2
 updates:
   - package-ecosystem: "npm"
+    versioning-strategy: "bump"
```

Comment:
Hm, I actually didn't see a strategy called `bump` in the docs. Maybe I missed something? https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#versioning-strategy

Reply:
omg thanks I mis-read this. 
Ok [looking at this again](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#versioning-strategy) I think it should be `increase`. Does that sound right to you?

Reply:
np! yeah, I think `increase` is what we want here

Reply:
done in 1b72dd4f9386d9d1e4a947ffc28f2199de29702e